kubeshop/monokle
# Ayush/feat/news feed

This PR closes the issue:
 - #3935 

## Changes

- Added news feed on right panel above 944px and at bottom below 944px
- Collapse of left panel at 1250px
- News feed updates at start only
- Added a new newsfeed model and reducer to set the news feed only at the start of the app. Everytime the startpage is opened again, the feed is fetched from the state and not via API call. API call is made only the initial start time.

## How to test it

- Run the electron app and in ```StartPage.tsx```, add a ```console.log("API function is called")``` just below where the API call is made. And then route to different panels and cluster modes. You will see that API call will only be made the initial time.
- Also try to resize the window to different widths to check the UI

## Screenshots

https://github.com/kubeshop/monokle/assets/72748253/8cb42e32-9adf-4fa6-91da-2c51691e3175



## Checklist
